recipes/scrapy/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I don't think the `bld.bat` file is required if you are using the `build: script:` section of the `meta.yaml`. That said, the recipe from Continuum uses separate `build.sh` and `bld.bat` files to delete some cryptography thing that I'm not 100% sure about:

https://github.com/ContinuumIO/anaconda-recipes/blob/master/scrapy/build.sh
https://github.com/ContinuumIO/anaconda-recipes/blob/master/scrapy/bld.bat